client/src/main/java/tds/assessment/Segment.java

```diff
@@ -143,7 +168,7 @@ public void setFieldTestMinItems(int fieldTestMinItems) {
     /**
      * @return the maximum number of field test items in the {@link tds.assessment.Segment}
      */
-    public int getFieldTestMaxItems() {
+    public Integer getFieldTestMaxItems() {
```

Comment:
Cool.  The point I was trying to make with the link I provided was that if I followed the code correctly the mapper was autoboxing them to primitives anyway so I don't think the code as written would ever allow null regardless of hte object type here.